app/assets/javascripts/components/dashboard/create_task.jsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * This is a test component to demonstrate modal configuration
```

Comment:
List any props (if any) you pass into the React component up here just for documentation purposes. See https://github.com/calblueprint/sjc/blob/master/app/assets/javascripts/components/ClientComments.jsx for an example

Reply:
I'm not sure what props `create_task` should take in yet, since the current `createTask` component might be outdated. let's put this on hold for now?